db/default_messages/overdue_email

```diff
@@ -1,10 +1,5 @@
-Dear @user@,\n\n\
-  It looks like you have overdue equipment!\n\n\
-  Please return the following equipment to us as soon as possible. Until then you will be charged a daily late fee of @late_fee@.\n\n\
-  @equipment_list@\n\n\
-  Failure to return equipment will result in the levying of replacement fees, and potential revocation of borrowing privileges.\n\n\
-  Your reservation number is Reservation_id@.\n\n\
-  Thank you,\n\
-  '@department_name@'
-
-
+It looks like you have overdue equipment!
+Please return the following equipment to us as soon as possible. Until then you will be charged a daily late fee of @late_fee@.
```

Comment:
Does this need to have an extra line break? In other words, don't we need to replace `\n\n` with two line breaks instead of one?